Updated switch topic (#412)
IgniteUI/indigo-design-docfx: en/components/switch.md

<img class="responsive-img" src="../images/switch_demo.png" srcset="../images/switch_demo@2x.png 2x" />

## Interaction State

The Switch can be inserted in an enabled or disabled state.

<img class="responsive-img" src="../images/switch_enabled&disabled-state.png" srcset="../images/switch_enabled&disabled-state@2x.png 2x" />

## Theme

The Switch can be used styled in **dark** and light variants to assure good readability and contrast for both lighter and darker backgrounds.

<img class="responsive-img" src="../images/switch_dark.png" srcset="../images/switch_dark@2x.png 2x" />
<img class="responsive-img" src="../images/switch_light.png" srcset="../images/switch_light@2x.png 2x" />

## Layout Template

> The Switch supports a label position where the Switch label is placed after or before the Switch element.

Reviewer: ```suggestion
The Switch supports a label position where the Switch label is placed before or after the Switch element.
```

## Styling

> The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as change of the label text style color. There is a fixed alpha value applied to the track to make it semi transparent.

Reviewer: ```suggestion
The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.```

## Styling

> The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.```

Reviewer 2: ```suggestion
The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.
```
